packages/fluent-theme/src/components/sendbox/Sendbox.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { type MouseEventHandler, useCallback, useRef, useState } from 'react';
+import cx from 'classnames';
+import { hooks } from 'botframework-webchat-api';
+import type { DirectLineCardAction, WebChatActivity } from 'botframework-webchat-core';
+import { SendIcon } from '../../icons/SendIcon';
+import { ToolbarButton, ToolbarSeparator, Toolbar } from './Toolbar';
+import { SuggestedActions } from './SuggestedActions';
+import { AddAttachmentButton, AttachmentDropzone, Attachments } from './Attachments';
+import { TextArea } from './TextArea';
+import { DialpadIcon } from '../../icons/DialpadIcon';
+import { useStyles } from '../../styles';
+
+const { useLocalizer } = hooks;
+
+const styles = {
+  webchat__sendbox__root: {
+    fontFamily: 'var(--fontFamilyBase)',
+    color: 'var(--colorNeutralForeground1)',
+    textRendering: 'optimizeLegibility'
+  },
+
+  webchat__sendbox__sendbox: {
+    fontFamily: 'var(--fontFamilyBase)',
+    padding: '8px',
```

Comment:
```suggestion
    padding: 8,
```

Reply:
Let's keep the units everywhere. They don't hurt and even help as:\
```
lineHeight: 1, // actual line height
padding: '2px 4px', // still needs to be a string
```

By simply not omitting units we avoid unnecessary ambiguity